packages/Makefile

```diff
@@ -21,6 +21,9 @@ all: .artifacts/bin/pyodide-build
 	mkdir -p $(PYODIDE_LIBRARIES)
 	$(HOSTPYTHON) -m pip install -e ../pyodide-build --no-deps --prefix $(PYODIDE_LIBRARIES)
 
+.artifacts/bin/wheel:
+	git clone git@github.com:hoodmane/wheel.git --branch handle-files-with-commas --depth 1 .artifacts/bin/wheel
```

Comment:
Maybe add some comment here to tell why we use custom fork of wheel and when it will be fixed?

Reply:
Right. It looks like my PR will be merged soon probably:
https://github.com/pypa/wheel/pull/427

Reply:
Okay I can actually just use `find -name '*,*' -delete`. Somehow I never think of the simple solution until after I've implemented a complicated solution =P